app/src/main/res/layout/fragment_home.xml

```diff
@@ -9,69 +9,35 @@
     android:background="@color/backgroundPrimary">
 
     <com.google.android.material.appbar.AppBarLayout
-        android:id="@+id/appBarLayout"
+        android:id="@+id/app_bar_layout"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:background="@color/backgroundPrimary"
         app:elevation="0dp">
 
         <com.google.android.material.appbar.CollapsingToolbarLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:elevation="0dp"
-            app:contentScrim="@color/backgroundPrimary"
             app:expandedTitleGravity="top"
             app:expandedTitleMarginEnd="48dp"
             app:expandedTitleMarginStart="48dp"
-            app:layout_scrollFlags="scroll|snap|exitUntilCollapsed">
+            app:layout_scrollFlags="scroll|exitUntilCollapsed">
 
-            <LinearLayout
+            <include
+                android:id="@+id/balance"
+                layout="@layout/layout_home_balance"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:background="@color/backgroundPrimary"
-                android:orientation="vertical">
+                android:layout_marginTop="80dp"
```

Comment:
it is better to add 80dp as toolbarSize
and use it here and in height of layout_home_toolbar